.circleci/config.yml

```diff
@@ -3355,6 +3355,14 @@ workflows:
               parallelism: [2]
               cluster-id-prefix: ["nightly"]
               mark: ["nightly"]
+      - test-e2e-aws:
```

Comment:
Right now this is only under `nightly` run, should we have a copy of this under `test-e2e-longrunning`? See: https://github.com/determined-ai/determined/pull/7520/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R3002
I guess it isn't required for PRs, but in my opinion `nightly` is not very visible for devs.